common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -47,6 +47,7 @@
     private boolean autoCreateSubscriptionGroup = true;
     private String messageStorePlugIn = "";
 
+    //thread counts for sending message, if large number threads are needed, we suggest setting useReentrantLockWhenPutMessage=true(in MessageStoreConfig)
```

Comment:
@shroman 

This is to make hints to developer after discuss with @zhouxinyu , since if developer hopes to change the sendNums, which may means propobly very feriece race conditions which by default using spin lock is not quite good enough. 

Accoring to the current spin lock implementations, the cpu will be runnig all the time without any sleep when trying to get spin lock, which may cause cpu problem when they set sendNums to a very large number with high concurrent message writing.

The comment to pointout that.